src/common/low_precision_transformations/src/convolution.cpp

```diff
@@ -333,6 +334,11 @@ bool ConvolutionTransformation::transform(TransformationContext &context, ov::pa
     ov::copy_runtime_info({ convolution, finalDequantization }, finalDequantization);
     updateOutput(context, finalDequantization, convolution);
 
+    const auto onActiviation = convolution->get_input_node_shared_ptr(0);
+    if (ov::is_type<ov::opset1::Subtract>(onActiviation)) {
+        DisableCleanupAttribute::create(onActiviation);
```

Comment:
We could move it to the [code section](https://github.com/openvinotoolkit/openvino/blob/c7bd2b796afb8f94491b04f0e54d94c43d4bca53/src/common/low_precision_transformations/src/convolution.cpp#L282-L300) where subtract on activations is handled to avoid additional checks

Reply:
Thanks for comment, the thought is clear, but the [code section](https://github.com/openvinotoolkit/openvino/blob/c7bd2b796afb8f94491b04f0e54d94c43d4bca53/src/common/low_precision_transformations/src/convolution.cpp#L282-L300) is connected with weights, but zero-point on activation is handled (zero index, not on weights). This is exactly the reason why the final zero-point is handled at the end of the method. : ))))

It's not so obvious to find where zero-point on activations is finally defined and can can be changed in the future. I prefer to handle it explicitly at the end. What do you think about that?